pkg/simulator/terraform.go

```diff
@@ -116,7 +123,7 @@ func Status(logger *zap.SugaredLogger, tfDir, bucket, attackTag string) (*Terraf
 	}
 
 	logger.Info("Running terraform output")
-	out, err := Terraform(tfDir, "output")
+	out, err := Terraform(tfDir, "output", bucket)
```

Comment:
I hate that we have to thread everything through like this :(  but that's my fault fo not anticipating that we'd need to change the initialisation parameters frequently.  I need to refactor